### ISSUE TITLE
Microphone and Audio not working 

### ISSUE THREAD

**Issue by aliabd** [gradio-app/gradio#1962]
### Describe the bug

Recording from microphone or clicking an audio example doesn't seem to be working in the latest version of gradio. 

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

This space: https://huggingface.co/spaces/gradio/same-person-or-different with the latest version of gradio. 

### Logs

```shell
Traceback (most recent call last):
  File "/Users/aliabdalla/anaconda3/envs/new/lib/python3.8/site-packages/gradio/routes.py", line 255, in run_predict
    output = await app.blocks.process_api(
  File "/Users/aliabdalla/anaconda3/envs/new/lib/python3.8/site-packages/gradio/blocks.py", line 630, in process_api
    predictions, duration = await self.call_function(fn_index, processed_input)
  File "/Users/aliabdalla/anaconda3/envs/new/lib/python3.8/site-packages/gradio/blocks.py", line 545, in call_function
    prediction = await anyio.to_thread.run_sync(
  File "/Users/aliabdalla/anaconda3/envs/new/lib/python3.8/site-packages/anyio/to_thread.py", line 31, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "/Users/aliabdalla/anaconda3/envs/new/lib/python3.8/site-packages/anyio/_backends/_asyncio.py", line 937, in run_sync_in_worker_thread
    return await future
  File "/Users/aliabdalla/anaconda3/envs/new/lib/python3.8/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/Users/aliabdalla/anaconda3/envs/new/lib/python3.8/site-packages/gradio/interface.py", line 489, in <lambda>
    lambda *args: self.run_prediction(args)[0]
  File "/Users/aliabdalla/anaconda3/envs/new/lib/python3.8/site-packages/gradio/interface.py", line 669, in run_prediction
    prediction = self.fn(*processed_input)
  File "app.py", line 52, in similarity_fn
    wav1, _ = apply_effects_file(path1, EFFECTS)
  File "/Users/aliabdalla/anaconda3/envs/new/lib/python3.8/site-packages/torchaudio/sox_effects/sox_effects.py", line 285, in apply_effects_file
    raise RuntimeError("Failed to load audio from {}".format(path))
RuntimeError: Failed to load audio from /var/folders/hb/p8tgjmhs0m19d_bgsjqv39nw0000gn/T/audiogg01jvg3.wav
```

### System Info

```shell
3.1.4
```

### Severity

blocking upgrade to latest gradio version

**Comment by freddyaboulton** [comment]
Should be fixed when the 3.2 release goes out. I'm assigning myself so that I don't forget to update the space.

**Comment by freddyaboulton** [comment]
Closing as this was fixed in #1987 !!
